android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -497,9 +497,14 @@ public View getView(int i, View view, ViewGroup viewGroup) {
       } else {
         setText(view, R.id.url, pwsResult.getSiteUrl());
       }
+      if (Utils.isResolvableDevice(pwPair.getUrlDevice())) {
+        ((ImageView) view.findViewById(R.id.icon)).setImageBitmap(
+            Utils.getBitmapIcon(mPwCollection, pwsResult));
+      } else {
+        ((ImageView) view.findViewById(R.id.icon))
+            .setImageResource(R.drawable.unresolved_result_icon);
```

Comment:
Can you just use (and shade) R.drawable.ic_lock_open_wht_24dp and R.drawable.ic_lock_outline_wht_24dp?